mono/mini/debugger-agent.c

```diff
@@ -7825,6 +7841,7 @@ assembly_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
         error_init (error);
         MonoClass* mono_class = mono_class_get_checked (ass->image, token, error);
         if (!is_ok (error)) {
+            buffer_add_string (buf, error->full_message);
```

Comment:
```suggestion
            buffer_add_string (buf, mono_error_get_message (error));
```

`error->full_message` is not always the field with the complete message (see `mono_error_get_message` for details)